### ISSUE TITLE
EarthCube Product Registry

### ISSUE THREAD

**Issue by 7yl4r** [marinebon/www_marinebon2#5]
I met with Stephen Richard from the EarthCube resource registry & USGIN to discuss potential synergy with EarthCube's efforts to catalog tools & tool metadata with regards to our #4. My dream would be to register our tools with the EarthCube registry and then create a pretty view using their backend or some RESTful frontend they would provide. 

Regarding registering: we should probably do that anyway. Here is the [EarthCube Tool Registry Form](https://docs.google.com/forms/d/e/1FAIpQLSfqyga5GGjKNB9CeQjO0kdIDaQ38A5nINXwBc-lmTTozMmP3w/viewform). Stephen was able to look at the seascapeR package and add it relatively easily. The resulting record can be viewed in the [Suave tool browser interface](https://suave2.sdsc.edu/main/file=zaslavsk_EC_Resource_Registry.csv), just search for "seascape". After another harvesting/processing step is complete the record will be viewable in the [GEOCODES tool browser](https://beta.geocodes.earthcube.org/landing.html). This tool browser will soon be upgraded to something like the [new GEOCODES interface (currently beta)](https://beta.geocodes.earthcube.org/landing.html).

My quick notes about way this all works:

1. registry form goes to spreadsheet
2. data gets picked up and turned into JSON-LD (based on Schema.org?).
3. data is put into a triplestore database so SPARQL queries can be run
4. other harvesters (like google datasets) can pick up the data published as JSON-LD using schema.org standards.

### In Summary
1. It would be ideal if we could leverage this existing infrastructure but a lot of work will need to be done. 
2. Stephen's group may be able to provide us with more features as they continue development. So this is worth keeping an eye on.
3. At a minimum we should consider the existing tool metadata work here
4. we should register our tools in the EC Registry either way

**Comment by bbest** [comment]
Nice! Here's what I'm seeing on desktop in [Suave tool browser interface](https://suave2.sdsc.edu/main/file=zaslavsk_EC_Resource_Registry.csv):

![image](https://user-images.githubusercontent.com/2837257/113898624-e3e55e00-9780-11eb-8f8e-1a332fcdf47d.png)

That's a good idea to borrow from fields there:

**seascapeR**
seascapeR
Resource type: Software
The seascapeR package provides functions for fetching, analyzing and visualizing Seascapes, a global and regional dynamic sea water classification product derived from satellite imagery by Maria Kavanaugh (OSU). MBON Seascapes identifies spatially ex...
links: https://marinebon.org/seascapeR/

Type of Resource
Software
Name
seascapeR
Description
The seascapeR package provides functions for fetching, analyzing and visualizing Seascapes, a global and regional dynamic sea water classification product derived from satellite imagery by Maria Kavanaugh (OSU). MBON Seascapes identifies spatially explicit water masses with particular biogeochemical features using a model and satellite-derived measurements. Marine Biodiversity Observation Network (MBON) is a global initiative to integrate data to improve understanding of changes and connections between marine biodiversity and ecosystem functions. In particular, this package is meant to help nodes in the MarineBON.org network extract Seascape data, especially across NOAA Sanctuaries, for comparison with biological data from eDNA, sound, telemetry and other observational data to evaluate how dynamic water masses relate to ecosystem function.
Citation
Best, Ben, Herrera, Enrique Montes, and Kavanaugh, Maria, 2021, SeascapeR,
Responsible Party
creator, person, Ben Best,
creator, person, Enrique Montes Herrera
creator, person, Maria Kavanaugh,
Readable Page
https://marinebon.org/seascapeR/
Keywords
oceanography, sea water mass classification
Maturity/status
In production
License
MIT
Target Audience
Data Users, Scientists, Members of the Public
Expected Lifetime
Unknown
Usage
Unknown
Related Resources
Seascapes classification scheme,
Science Domain
Biological Oceanography, Chemical Oceanography, Environmental Science, Marine Geology
Funding Source
US National Oceanic and Atmospheric Administration (NOAA)
Programming Languages
R
Code Repository URL
https://github.com/marinebon/seascapeR
Input format
text/plain;application=OpenDAPv2-DDS
Runtime Environment
R studio
Installer Package
In R Studio remotes::install_github('marinebon/seascapeR')
Stewardship Name
Ben Best,
Stewardship Type
Person
Registry Contributor
Stephen M. Richard
ORCID of the Contributor
https://orcid.org/0000-0001-6041-5302
Email Address
smrTucson@gmail.com
Timestamp
Fri Apr 02 2021 13:11:29 GMT-0700 (Mountain Standard Time)

**Comment by bbest** [comment]
Also, there's  more on how to enable dataset discovery here:  [Dataset  |  Search Central  |  Google Developers](https://developers.google.com/search/docs/data-types/dataset)
